docs/building-on-etherlink/development-toolkits.md

```diff
@@ -8,9 +8,41 @@ title: Development toolkits
 
 ### Using Hardhat with Etherlink
 
-It is recommended to use the following config in your `hardhat.config.js` file:
+Hardhat works seamlessly with Etherlink.
+You can follow the documentation at https://hardhat.org/docs and insert values for Etherlink.
 
+Here is an example `hardhat.config.js` file for Etherlink Mainnet:
+
+```javascript
+module.exports = {
+  solidity: "0.8.24",
+  networks: {
+    etherlinkMainnet: {
+      url: "https://node.mainnet.etherlink.com",
+      accounts: ["<YOUR_PRIVATE_KEY>"],
+    }
```

Comment:
Can we have both mainnet and testnet for the config. Same for etherscan api keys and custom chains?

```
etherlinkTestnet: {
      url: "https://node.ghostnet.etherlink.com",
      accounts: ["<YOUR_PRIVATE_KEY>"],
    }
    ```

Reply:
Ok, I merged the config files into one.